pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -20,6 +20,7 @@ type LifecycleProcessor struct {
 	ProcessTrace        func(p *api.Payload)
 	Demux               aggregator.Demultiplexer
 	DetectLambdaLibrary func() bool
+	SamplingRate        float64
```

Comment:
Nit: Should this be an `int32`? That is what trace chunks use and the value can only be 0, 1, or 2 as far as I'm aware.